developer-docs/plugin-guide.md

```diff
@@ -35,84 +36,37 @@ Pupil Player uses an identical plugin structure. Little (often no work) needs to
 
 ### Plugin API
 
-Plugins inherit the [`Plugin` class](https://github.com/pupil-labs/pupil/blob/v0.9.12/pupil_src/shared_modules/plugin.py#L23). It provides default functionality as well as series of callback functions that are called by the world process. [The source](https://github.com/pupil-labs/pupil/blob/v0.9.12/pupil_src/shared_modules/plugin.py#L23) contains detailed information about the use-cases of the different callback functions.
+Plugins are Python classes that inherit the [`Plugin` class](https://github.com/pupil-labs/pupil/blob/568604bd06a972eeac533f74079c22459b6e929e/pupil_src/shared_modules/plugin.py#L23). It provides default functionality as well as series of callback functions that are called by the world process. [The source](https://github.com/pupil-labs/pupil/blob/568604bd06a972eeac533f74079c22459b6e929e/pupil_src/shared_modules/plugin.py#L23) contains detailed information about the use-cases of the different callback functions.
 
 <aside class="warning">
 Plugin callbacks are called regularly within the main thread. If your custom plugin uses functions that block longer than ~0,02 seconds it will lead to unresponsive ui and the framerate will drop.
 </aside>
 
-#### Make your own plugin
-These general steps are required if you want to make your own plugin and use it within Pupil:
+#### Register your plugin automatically
+Plugins are registered on application start. System plugins are [registered explicitly](https://github.com/pupil-labs/pupil/blob/618181d157bfc3e75415a305c6f89c90ac0e614f/pupil_src/launchables/world.py#L116-L139).
```

Comment:
Ok, I see what you're doing. You are referencing the latest commit for this file.

Let's leave this as you have it for now, but we should update this link to the associated Pupil tag before we tag the docs.